src/layouts/MainLayout.vue

```diff
@@ -162,6 +163,19 @@ export default {
     } catch (err) {
       console.error(err)
     }
+  },
+  mounted () {
+    this._keyListener = function(e) {
+      if (e.key === "k" && (e.metaKey || e.ctrlKey)) {
+        e.preventDefault()
+        this.toggleContactBookOpen()
```

Comment:
I'd recommend making this a bound prop.  It's best to avoid Vuex as it fires a bunch of events all over the place. You can check out the `contactDrawerOpen` stuff.  You can also just add another q-dialog to MainLayout with appropriate handlers.

Reply:
I'll try that

Reply:
I updated the rest of the code but I'm not sure what you mean here by "bound prop".